.github/workflows/update-copilot-env.yml

```diff
@@ -0,0 +1,65 @@
+name: Update the copilot environment
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description:  Which AWS Account to use?
+        type: choice
+        required: true
+        options:
+        - test
+      workspace:
+        description:  Which Copilot Application to use?
+        type: choice
+        required: true
+        options:
+        - pre-award
+        - post-award
+  push:
+    paths: # Ignore README markdown and only deploy when something in the copilot folder has changed
+      - 'copilot/**'
+      - '!**/README.md'
+  pull_request:
+    types:
+      - closed # Further protection - only allow this workflow to run automatically on closed pull requests
+
+jobs:
+  copilot_build:
+    if: github.event.pull_request.merged == true
+    concurrency: deploy-copilot-env-${{ inputs.environment || 'test' }}
+    permissions:
+      id-token: write # This is required for requesting the JWT
+      contents: read  # This is required for actions/checkout
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment || 'test' }}
+    steps:
+    - name: Git clone the repository
+      uses: actions/checkout@v3
+
+    - name: Get current date
+      id: currentdatetime
+      run: echo "datetime=$(date +'%Y%m%d%H%M%S')" >> $GITHUB_OUTPUT
+
+    - name: configure aws credentials
+      uses: aws-actions/configure-aws-credentials@v2
+      with:
+        role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT }}:role/GithubCopilotDeploy
+        role-session-name: COPILOT_ENV_${{ steps.currentdatetime.outputs.datetime }}
+        aws-region: eu-west-2
+
+    - name: Install AWS Copilot CLI
+      run: |
+        curl -Lo aws-copilot https://github.com/aws/copilot-cli/releases/latest/download/copilot-linux && chmod +x aws-copilot && sudo mv aws-copilot /usr/local/bin/copilot
+
+    - name: Set the workspace
+      run: |
+        touch copilot/.workspace
+        echo application: ${{ inputs.workspace || 'pre-award' }} >> .workspace
```

Comment:
Is this still required if you specify the env in the later command?

Reply:
I'm a little confused what you mean? The pre-award is a default value incase there isn't a pre-award application.

Setting the application in the workspace is a requirement of copilot - it'll assume there's no app otherwise

Reply:
With the new folder structure, could we hardcode this file instead?

Reply:
...you make an excellent point. Yes, will fix that up :D